amlip_demo_nodes/sync/sync_main_node.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
```suggestion
# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Reply:
This is not a new file, but a moved one:
https://github.com/eProsima/AML-IP/pull/77#discussion_r1221317673